tests/gis_tests/gdal_tests/models.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.gis.db import models
+from django.db import connection
+
+HAS_POSTGIS_2 = (
+    hasattr(connection.ops, 'postgis') and
```

Comment:
There was one, but not anymore. I updated the test to use the feature flag  and moved the import of the test model into the test. That solved the problem. Thanks!